ingress/ingress-rules-goquorum.yml

```diff
@@ -28,11 +28,11 @@ spec:
   rules:
   - http:
       paths:
-      - path: /rpcnode
+      - path: /rpc
         pathType: Prefix
         backend:
           service:
-            name: goquorum-node-rpcnode-1
+            name: goquorum-node-rpc-1
```

Comment:
`helm install rpc-1 ./charts/goquorum-node --namespace quorum --values ./values/reader.yml`

This change is to bring it in line with following command